test/run.d

```diff
@@ -250,47 +329,35 @@ string[string] getEnvironment()
     string[string] env;
 
     env["RESULTS_DIR"] = resultsDir;
-    auto os = env.getDefault("OS", detectOS);
-    auto build = env.getDefault("BUILD", "release");
+    env["OS"] = os;
+    env["MODEL"] = model;
+    env["BUILD"] = build;
```

Comment:
Hmm, I would rename these three to make clear to the reader where they come from.
How about `targetOS` or `buildOS`? Or maybe simply doing all UPPERCASE would work?
I know that's a bit against the style guide, but this comes from the Makefiles where they are uppercased.

Reply:
Is it relevant where they come from? It was called `os` before:

https://github.com/dlang/dmd/blob/a60e9e25f075af0179f521e8e6864d892996a713/test/run.d#L253